internal/http/router.go

```diff
@@ -27,3 +27,9 @@ const EventLogRouterPath = "/eventlog"
 
 // ExprPath is path for expression.
 const ExprPath = "/expr"
+
+// TriggerStopPath is path for trigger component stop
+const TriggerStopPath = "/stop"
```

Comment:
suggest moving all operations of cluster management to /management root path,  we should be cautious while add the kind of path under root path